.travis/BuildLinux.sh

```diff
@@ -57,12 +57,22 @@ if [ -z "${RUN_CLANG_TIDY}" ] \
     && [ -z "${RUN_IWYU}" ] \
     && [ -z "${BUILD_DOC}" ]; then
     if [[ ${TRAVIS_BUILD_STAGE_NAME} = "Build libraries" ]]; then
-        time make libs test-libs-stage1 -j2
+        time make libs -j2
+        time make test-libs-domain -j2
```

Comment:
Why not use the `test-libs-stage1` target here?

Reply:
I thought it would be more interesting to track the timing of smaller groups of test libraries

Reply:
Okay, then can we remove the `test-libs-stage1` target?